### ISSUE TITLE
Save packages in a proper folder organization so we could use it as a local repo later

### ISSUE THREAD

**Issue by charlesprost** [scality/metalk8s#615]
Blocked by #587 

Instead of downloading/building all packages in one single location, let's check if we need to build a proper folder organization (could be a tree of folders) so we could use the top folder as a local repo.

Also provide any configuration file needed for the local repo.

The local repo creation will be performed in another ticket.

**Comment by alexandre-allard** [comment]
For now, repositories tree is located under _build/root/packages and looks like this:
```
# ls _build/root/packages/
base-el7/  docker-ce-el7/  extras-el7/  kubernetes-el7/  saltstack-el7/  scality-el7/  updates-el7/
```
One repository per source repo.

**Comment by charlesprost** [comment]
@alexandre-allard-scality so we can close this one I guess?

**Comment by alexandre-allard** [comment]
We still need to generate a configuration file for the local repos to be used on the deployment hosts, but it's no longer related to the build part.
So, yes, I think we can close this one as soon as my PR related to #587 is merged.
